modules/default/compliments/compliments.js

```diff
@@ -42,43 +42,57 @@ Module.register("compliments", {
 		mockDate: null,
 		advice: false
 	},
-	lastIndexUsed:-1,
+	lastIndexUsed: -1,
 	// Set currentweather from module
 	currentWeatherType: "",
 
 	// Define required scripts.
-	getScripts: function() {
+	getScripts: function () {
 		return ["moment.js"];
 	},
 
 	// Define start sequence.
-	start: function() {
+	start: function () {
 		Log.info("Starting module: " + this.name);
 
 		this.lastComplimentIndex = -1;
 
 		var self = this;
 		if (this.config.remoteFile !== null) {
-			this.complimentFile(function(response) {
+			this.complimentFile(function (response) {
 				self.config.compliments = JSON.parse(response);
 				self.updateDom();
 			});
-		} else if (this.config.advice) {
+		}
+
+		if (this.config.advice) {
 			var xobj = new XMLHttpRequest();
```

Comment:
that could be a const too, right?

Reply:
Yes it absolutely could. Just updated ;)